### PR TITLE
Packet().overload_fields can be overwritten

### PR DESCRIPTION
**Pull request by p-l-**
This is needed by fragment(), which has been broken since PR #11.

### CODE REVIEW

**Comment by p-l-** [comment]
Before:
```python
>>> fragment(IP() / ("X" * 10), 8)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "scapy/layers/inet.py", line 766, in fragment
    r.overload_fields = p[IP].payload.overload_fields.copy()
  File "scapy/packet.py", line 217, in __setattr__
    except AttributeError:
AttributeError: 'Raw' object attribute 'overload_fields' is read-only
```

After:
```python
>>> fragment(IP() / ("X" * 10), 8)
[<IP  flags=MF frag=0 |<Raw  load='XXXXXXXX' |>>, <IP  flags= frag=1 |<Raw  load='XX' |>>]
```

**Comment by guedou** [comment]
I think that it would be a good idea to addd this call to fragment to the unit tests.

**Comment by p-l-** [comment]
@guedou agreed! Plus that allowed me to notice that `defragment()` was broken, too :)

File: scapy/base_classes.py [hunk 1]
```diff
@@ -180,8 +180,13 @@ def __new__(cls, name, bases, dct):
 
         if "__slots__" not in dct:
             dct["__slots__"] = []
-        if "name" in dct:
-            dct["_name"] = dct.pop("name")
+        for attr in ["name", "overload_fields"]:
+            try:
+                dct["_%s" % attr] = dct.pop(attr)
+                if cls.__name__ == "ISAKMP_payload_SA":
+                    print dct["_%s" % attr]
```
Comment (guedou) [inline scapy/base_classes.py]: I guess that this is a debugging statement that was forgotten =)

Comment (p-l-) [inline scapy/base_classes.py]: Hum. Thanks :)

**Pull request merged**